src/prefect/cli/run.py

```diff
@@ -147,7 +165,27 @@ def cloud(
     flow_run_id = client.create_flow_run(
         flow_id=flow_id, parameters={**file_params, **string_params}, run_name=run_name
     )
-    click.echo("Flow Run ID: {}".format(flow_run_id))
+
+    if flow_run_id_only:
+        click.echo("Flow Run ID: {}".format(flow_run_id))
+    else:
+        # Generate direct link to Cloud run
+        tenant_slug = client.get_default_tenant_slug()
+
+        url = (
+            re.sub("api-", "", config.cloud.api)
+            if re.search("api-", config.cloud.api)
+            else re.sub("api", "cloud", config.cloud.api)
+        )
+
+        def join(url: str, items: list) -> str:
+            if len(items) == 1:
+                return urljoin(url, items[0])
+            return join(urljoin(url, items[0]) + "/", items[1:])
```

Comment:
I definitely get this, but since we know that the config URL won't include any terminal pages (e.g., `index.html`) I suspect we can get away with a simpler implementation here, e.g., 
```python
"/".join([url.rstrip("/"), tenant_slug, "flow-run", flow_run_id])
```